docs/Getting_Started.md

```diff
@@ -1,47 +1,14 @@
-# Prerequisites 
+# Overview
```

Comment:
It seems a little strange to not have any text below this title. Should we add a blurb here about a generic "Getting Started" to fill out this page more?

Reply:
@bradleynull , I am not sure if this is enough, but I want to keep it simple. How does this look? https://monodrive.readthedocs.io/en/getting_started_updates/Getting_Started/